docs-mslearn/fabric/create-finops-workspace.md

```diff
@@ -0,0 +1,235 @@
+
+## Export cost data
+
+You can use existing actual or amortized cost exports; however, the following example creates a new cost and usage details (FOCUS) data export.
+
+What is FOCUS? FinOps Open Cost and Usage Specification (FOCUS) is a new, cross-cloud billing data standard hosted by the FinOps Foundation and defined by Microsoft and a collection of FinOps practitioners, cloud providers, and vendors. To learn more about FOCUS and its benefits, see [About FOCUS](https://microsoft.github.io/finops-toolkit/focus). The FinOps Foundation also offers a free [Introduction to FOCUS course](https://learn.finops.org/introduction-to-focus). Microsoft’s Power BI solutions for FinOps are aligned to FOCUS.
+
+To create an export, see [Create exports](/azure/cost-management-billing/costs/tutorial-improved-exports#create-exports).
+
+Here are the high-level steps to create an export:
+
+1. To navigate:
+    1. Sign in to the Azure portal at [https://portal.azure.com](https://portal.azure.com/), search for **Cost Management**.
+    2. Select the required scope and select **Exports** in the left navigation menu.
+    3. Select **+ Create**
+2. On the Basics tab:
+    1. Select the template = **Cost and usage (FOCUS)**
+        :::image type="content" source="./media/create-fabric-workspace-finops/exports-cost-and-usage-focus.png" border="true" alt-text="Screenshot showing creating a new FOCUS export dataset." lightbox="./media/create-fabric-workspace-finops/exports-cost-and-usage-focus.png" :::
+3. On the Datasets tab:
+    1. Fill in **Export prefix** to ensure you have a unique name for the export. For example, *june2024focus*.
+4. On the Destination tab:
+    1. Storage type = Azure blob storage
+    2. Destination and storage = **Use existing**
+    3. Subscription = *Contoso subscription*
+    4. Storage account = *Contoso-storage-account*
+    5. Container = *msexports*
+    6. Directory = focuscost/&lt;*scope-id*&gt;
+    7. Overwrite data = **Enabled**
+5. On the Review + Create tab:
+    - Select **Create**
+
+Ensure that your export completed and that the file is available before moving to the next step.
+
+>[!IMPORTANT]
+> Cost Management exports utilize managed identity. To create an export, you must be an Owner of the storage account or have access to grant permissions to the storage account. To see all required permissions, see [Prerequisites](/azure/cost-management-billing/costs/tutorial-improved-exports#prerequisites).
+
+If you want to automate export creation, consider using the [New-FinOpsCostExport command](https://microsoft.github.io/finops-toolkit/powershell/cost/New-FinOpsCostExport) in the FinOps toolkit PowerShell module.
+
+If you deployed [FinOps hubs](https://aka.ms/finops/hubs), you can use the exports configured against the hub.
+
+## Create a Fabric workspace
```

Comment:
Since we include this in the prerequisites, do we need a section for it? I'm fine either way. Just thinking out loud. If we remove this one, we should probably add the links to create one above in the prereqs section.

NOTE: The rest of my feedback for this section is assuming we keep it.

Reply:
I think we should keep the section in case people haven't already created a workspace. I intentionally worded the prereq to say:  "To complete this walkthrough, you create the following resources that incur costs: - A Fabric workspace "